File: take_admin_screenshot.py
#!/usr/bin/env python3
import asyncio
from playwright.async_api import async_playwright

async def main():
    async with async_playwright() as p:
        # Launch browser
        browser = await p.chromium.launch(headless=True)
        page = await browser.new_page()
        
        try:
            # Navigate to login page
            print("Navigating to login page...")
            await page.goto("http://localhost:5651/login")
            
            # Fill login form
            print("Filling login form...")
            await page.fill('input[type="email"]', 'admin@witchcityrope.com')
            await page.fill('input[type="password"]', 'Test123!')
            
            # Submit login form
            print("Submitting login...")
            await page.click('button[type="submit"]')
            
            # Wait for navigation to complete
            await page.wait_for_load_state('networkidle')
            
            # Navigate to admin dashboard
            print("Navigating to admin dashboard...")
            await page.goto("http://localhost:5651/admin/dashboard")
            
            # Wait for the page to fully load
            print("Waiting for page to load...")
            await page.wait_for_selector('text="Dashboard Overview"', state='visible', timeout=30000)
            
            # Take full page screenshot
            print("Taking screenshot...")
            await page.screenshot(path='admin-dashboard-current-state.png', full_page=True)
            
            print("Screenshot saved as admin-dashboard-current-state.png")
            
        except Exception as e:
            print(f"Error occurred: {e}")
            
        finally:
            # Close browser
            await browser.close()

if __name__ == "__main__":
    asyncio.run(main())

File: take_admin_screenshot.js
const { chromium } = require('playwright');

(async () => {
  // Launch browser
  const browser = await chromium.launch({
    headless: true // Run in headless mode
  });
  
  const context = await browser.newContext({
    viewport: { width: 1920, height: 1080 } // Set a good viewport size for screenshots
  });
  
  const page = await context.newPage();
  
  try {
    // Navigate to login page
    console.log('Navigating to login page...');
    await page.goto('http://localhost:5651/login', { waitUntil: 'networkidle' });
    
    // Fill in login credentials
    console.log('Filling login form...');
    await page.fill('input[name="Input.EmailOrUsername"]', 'admin@witchcityrope.com');
    await page.fill('input[name="Input.Password"]', 'Test123!');
    
    // Submit login form
    console.log('Submitting login...');
    await page.click('button[type="submit"]');
    
    // Wait for navigation to complete after login
    await page.waitForLoadState('networkidle');
    
    // Navigate to admin dashboard
    console.log('Navigating to admin dashboard...');
    await page.goto('http://localhost:5651/admin/dashboard', { waitUntil: 'networkidle' });
    
    // Alternative: try just /admin if /admin/dashboard doesn't work
    if (page.url() !== 'http://localhost:5651/admin/dashboard') {
      console.log('Trying /admin instead...');
      await page.goto('http://localhost:5651/admin', { waitUntil: 'networkidle' });
    }
    
    // Wait a bit for any dynamic content to load
    await page.waitForTimeout(2000);
    
    // Take full page screenshot
    console.log('Taking screenshot...');
    await page.screenshot({ 
      path: 'admin-dashboard-current.png',
      fullPage: true // Capture the entire page including scrollable content
    });
    
    console.log('Screenshot saved as admin-dashboard-current.png');
    
  } catch (error) {
    console.error('Error occurred:', error);
  } finally {
    // Close browser
    await browser.close();
  }
})();